test/martha/martha_v3.test.js

```diff
@@ -153,6 +165,52 @@ test.serial('martha_v3 uses the default error handler for unexpected errors', as
     sinon.assert.callCount(response.send, 0);
 });
 
+test.serial('martha_v3 calls the correct endpoints when only the accessUrl is requested with passports', async (t) => {
+    const {
+        id: objectId, self_uri: drsUri,
+        access_methods: { 0: { access_id: accessId, access_url: { url: gcsUrl } } }
```

Comment:
I think you want this:
```suggestion
        access_methods: [{ access_id: accessId, access_url: { url: gcsUrl } }]
```

Reply:
This is just keeping consistent with other code like this in this file, which I added in the first place. I can't believe I didn't try that syntax, which makes me want to double-check that it works... though surely you know better than I do so this is probably just a 🤦 moment.